static/js/chart/draw.ts

```diff
@@ -431,42 +436,77 @@ function drawLineChart(
 
 
 /**
- * Draw a group of line charts with different goeId.
+ * Return all styles information for given dataGroupsDict.
+ * {
+ *     colors: [],
+ *     dashes: [],
+ *     statVars: [],
```

Comment:
Please add some descriptions here, particularly for statVars and geoIds since it's not immediately clear how they relate to chart drawing styles.

Reply:
Re beets comments, since this function is returning more than styles, can rename this to computeStyleAndLegend(). Also can have a separate function to compute maxV, like getMaxValue()

Also you can specify the return value types in the function signature, example: https://www.typescriptlang.org/docs/handbook/functions.html#writing-the-function-type

so the function is something like:

function generateStyles(...) : {colors: string[], dashes: string[], ...., maxV: number}

Reply:
Done